src/apps/lwaftr/ndp.lua

```diff
@@ -2,10 +2,9 @@ module(..., package.seeall)
 
 -- NDP address resolution.
 -- Given a remote IPv6 address, try to find out its MAC address.
--- Exit if this fails.
 -- If resolution succeeds:
 -- All packets coming through the 'south' interface (ie, via the network card)
--- are silently forwarded.
+-- are silently forwarded (unless dropped by the network card).
```

Comment:
Nope - I added the text to make it clear to the reader that there were elements outside of it in the app network that could prevent packets from reaching this app, but I agree it's suboptimal wording. :-)